.github/workflows/ci.yml

```diff
@@ -13,27 +26,33 @@ jobs:
       # Install Tools
       - name: Install Tools
         run: |
-          sudo apt-get install wget build-essential python3 verilator libevent-dev libjson-c-dev device-tree-compiler
-          pip3 install setuptools
-          pip3 install requests
-          pip3 install pexpect
+          apt-get -qy update
```

Comment:
This is one of the steps that should not be repeated, because doing so does not add any value.

With regard to the specific list of packages, `python3` is installed already in all ghcr.io/hdl/debian-buster images. See https://github.com/hdl/containers/blob/main/debian-buster/base.dockerfile#L26.